atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/impl/TableRemappingKeyValueServiceTest.java

```diff
@@ -89,6 +96,144 @@ public void canConcurrentlyReadFromStableTablesWhileCreatingAndDroppingOtherTabl
         }
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {false}) // KvTableMappingService issues prevent cached lookups from working as intended
```

Comment:
A FLUP change can try setting cached=true and resolving more concurrency/caching problems.

Reply:
(Note that this same test actually breaks with caching disabled, prior to removal of the cached table lookup removal in this change)